src/Component/page/Reservations/Form.js

```diff
@@ -35,17 +36,20 @@ const Form = () => {
       <div className='d-grid two-col-grid my-1 grid-gap-1 text-sage'>
         <div>
         <label className='d-block'>Name</label>
-        <input className='form-input' type="text" />
+        <input className='form-input' type="text" id="name" name="name" required
+       minlength="4" maxlength="8" size="10" />
         </div>
         <div className='text-sage'>
         <label className='d-block'>Contact Number</label>
-        <input className='form-input' type="text" />
+        <input className='form-input' type="tel" id="phone" name="phone"
+       pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}"
```

Comment:
This regex isn't the best one, I'm afraid. We don't normally write UK phone numbers in this way (we usually write them with spaces or all as one number) so the user would find it confusing especailly as there is no error message. Phone number validation is a fiddly business. I would limit it to numbers only and display an error message, any more complicated validation could drive away customers